src/components/tooltip/Tooltip.tsx

```diff
@@ -96,6 +98,8 @@ export type TooltipProps = {
     offset?: string;
     /** Function called if the user manually dismisses the tooltip - only applies if showCloseButton is true */
     onDismiss?: () => void;
+    /** allow the component to maintain focus until escaped */
```

Comment:
smallest of all smalls
```suggestion
    /** Allow the component to maintain focus until escaped */
```
because the other comments are capitalized